### ISSUE TITLE
Rename/Rebrand 

### ISSUE THREAD

**Issue by coffandro** [MakovWait/godots#92]
Hello there, I absolutely love Godots but a slight concerns with the project is how similar it lies to Godot itself (enough that I misspelled while writing this). it already shows via [issues like the Flatpak possibly being pulled soon](https://github.com/flathub/io.github.MakovWait.Godots/issues/5) and I would just like to provide some possible options if the maintainer wanted to go down this route:
Godot Hub (like the one for unity, also makes it more clear what this project does exactly)
Godot Instance Manager (Boring name but describes what it is)
Godot Manager (Simpler than previous, still lays out purpose of project)

**Comment by MakovWait** [comment]
Hey! I would like to leave things as they are. 
Would it be valid option to introduce new branch/fork for Flathub version?

**Comment by coffandro** [comment]
Maybe? depends what the guys in the flatpak repo think. 
a secondary point I forgot to bring up is that Godots as a name isn't very searchable and dosen't really show up on google or whatever, was hard to find it when I tried to give it to some friends a while ago for instance.

**Comment by coffandro** [comment]
![Godots](https://github.com/MakovWait/godots/assets/75324436/81efa22f-0802-48ef-864a-edcfcedee7ab)
I've made this little thing as a possibility

**Comment by bbhtt** [comment]
>  Would it be a valid option to introduce a new branch/fork for Flathub version?

Yes this is fine as long as upstream is ok with it.

**Comment by kendallroth** [comment]
I would agree with this point as well; I do like the name "Godots" but agree that is was very hard to track down when I wanted to find again (until I remember to specifically search with a GitHub reference) 🤷.

> a secondary point I forgot to bring up is that Godots as a name isn't very searchable and dosen't really show up on google or whatever, was hard to find it when I tried to give it to some friends a while ago for instance.

**Comment by Croydon** [comment]
How about something like "Hub for Godot"?

I am not a lawyer and this is not legal advise, specifically since it is different in different countries anyway, however, I think that when you name "something for x" it is much more okay with trademark laws, even when you mention a registered trademark.

I believe this is the reason why Microsoft named it "Windows Subsystem for Linux" to not get in potential trouble with the Linux Foundation.

**Comment by athyfr** [comment]
> Godot Manager (Simpler than previous, still lays out purpose of project)

FYI, that name seems already taken by a [similar project.](https://github.com/eumario/godot-manager)
